news/13424.feature.rst

```diff
@@ -0,0 +1 @@
+Handle pathologically connected dependency graphs in reasonable time.
```

Comment:
Update this news item, it should be framed in the context of a pip user reading the changelog, e.g.

```
Minor performance improvement getting the order to install a very large number of interdependent packages.
```